Sources/SwifterSwift/AppKit/NSStackViewExtensions.swift

```diff
@@ -0,0 +1,26 @@
+// NSStackViewExtensions.swift - Copyright 2024 SwifterSwift
+
+#if canImport(AppKit) && !targetEnvironment(macCatalyst)
+import AppKit
+
+// MARK: - Methods
+
+public extension NSStackView {
+    /// SwifterSwift: Adds array of views to the end of the arrangedSubviews array.
+    ///
+    /// - Parameter views: views array.
+    func addArrangedSubviews(_ views: [NSView]) {
```

Comment:
Would this work? Is more generic.
```suggestion
    func addArrangedSubviews<S: Sequence>(_ views: S) where Element: NSView {
```

Reply:
I'm sure it would (except you need `S.Element`, not `Element`), however
1. we can keep at aligned with the `UIStackView` extension
2. I don't think we need to go all-out on using generic sequences _everywhere_. I think the most common use case here would be an `Array`.

Reply:
@LucianoPAlmeida Hello, I just took it from UIStackViewExtensions.swift.